Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -784,9 +803,9 @@ function logInfo {
 # In the past we copied to the users temp folder, difficult to find on machines with lots of profiles. Solved this by always placing the log in the normal Huntress folder.
 function copyLogAndExit {
     Start-Sleep 1
-    $logLocation = $DebugLog
-    $agentPath   = getAgentPath
-    Copy-Item -Path $logLocation -Destination $agentPath -Force
+    $agentPath = getAgentPath
+    $logLocation = Join-Path $agentPath "HuntressPoShInstaller.log"
+    Copy-Item -Path $Debuglog -Destination $agentPath -Force
```

Comment:
should be $DebugLog ?

Reply:
![image](https://user-images.githubusercontent.com/111775758/230502745-e3b33341-485e-4b20-aab6-90520910377e.png)
ok apparently powershell is just never case sensitive lol

Reply:
Nice save. Crying on the inside as it was `$Debuglog` in another place too (that wasn't me)

Reply:
I fixed it anyway